.github/workflows/client.yml

```
@@ -1,86 +1,83 @@
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: 12
 
-      - name: Cache node modules
-        uses: actions/cache@v1
+      - uses: actions/cache@v1
         with:
           path: client/node_modules
-          key: ${{ runner.OS }}-${{ matrix.node-version}}-node-modules-${{ hashFiles('client/yarn.lock') }}
+          key: ${{ runner.os }}-${{ hashFiles('client/yarn.lock') }}
 
-      - name: Install Dependencies
-        run: |
-          cd client
-          yarn install
-
-      - name: Build
-        run: |
-          cd client
-          yarn build
-
-  # Same cleaning need to be done, see Issue #105
-  #      - name: Lint
-  #        run: |
-  #          cd client
-  #          yarn lint
+      - run: yarn --cwd ./client install
+      - run: yarn --cwd ./client build
+      # Some cleaning need to be done, see Issue #105
+      # - run: yarn --cwd ./client lint
 
   publish:
-    name: Build and publish
-    env:
-      GITHUB_REGISTRY: docker.pkg.github.com
-      IMAGE: arkhn/pyrog-client
-    runs-on: ubuntu-18.04
     needs: build
-    if: github.ref == 'refs/heads/master'
-    strategy:
-      matrix:
-        environment:
-          [
-            { "name": "ambroise", "secret": "IP_VM_AMB" },
-            { "name": "aphp", "secret": "IP_VM_AP" },
-            { "name": "clb", "secret": "IP_VM_CLB" },
-            { "name": "curie", "secret": "IP_VM_CURIE" },
-            { "name": "demo", "secret": "" },
-            { "name": "dev", "secret": "" },
-            { "name": "rms", "secret": "IP_VM_REIMS" },
-            { "name": "staging", "secret": "" },
-          ]
-    steps:
-      - uses: actions/checkout@v1
+    runs-on: ubuntu-18.04
 
-      - name: Build image
-        env:
-          IP: ${{ secrets[matrix.environment.secret] }}
-          STAGE: ${{ matrix.environment.name }}
+    steps:
+      - id: metadata
         run: |
-          cd client
-          if [ -n "${{ matrix.environment.secret }}" ]; then
-            ./scripts/generate_client_env.sh > .env.${{ matrix.environment.name }}
-          fi
-          docker build -t $IMAGE:${{ matrix.environment.name }} --build-arg ENV_FILE=.env.${{ matrix.environment.name }} .
+          IMAGE=arkhn/pyrog-client
+          # Always tag with first 8 of sha
+          TAGS="${IMAGE}:${GITHUB_SHA::8}"
 
-      - name: Tag docker image with hash
-        run: |
-          docker tag $IMAGE:${{ matrix.environment.name }} $IMAGE:${{ matrix.environment.name }}-$GITHUB_SHA
+          # Has this workflow been triggered by an app tag ?
+          if [[ "${GITHUB_REF}" == refs/tags/app/* ]]; then
+            VERSION=${GITHUB_REF#refs/tags/app/}
+            # Is the tag on master and does it follow numeric versioning ?
+            if [[ "${{ github.event.base_ref }}" == refs/heads/master ]] && [[ "${VERSION}" =~ ^v[0-9]{1,3}(\.[0-9]{1,3}(\.[0-9]{1,3})?)?$ ]]; then
+              TAGS="${TAGS},${IMAGE}:latest"
```

I suggest we strip the leading `v` from this tag. It's standard for numerically versioned docker image.
Cf arkhn/warehouse-api/pull/160

```suggestion
              TAGS="${TAGS},${IMAGE}:latest"
              VERSION="${VERSION#v}"
```